lib/exirc/client.ex

```diff
@@ -853,7 +853,7 @@ defmodule ExIRC.Client do
     if state.debug?, do: debug "#{from} SENT #{message} TO #{to}"
     send_event {:received, message, sender, to}, state
     # If we were mentioned, fire that event as well
-    if String.contains?(message, nick), do: send_event({:mentioned, message, sender, to}, state)
+    if String.contains?(String.downcase(message), String.downcase(nick)), do: send_event({:mentioned, message, sender, to}, state)
```

Comment:
Just a quick note: This trick is enabled by the fact that IRC restricts us to ASCII for nicknames. Otherwise we might have experienced case conversion collision, as described here: https://eng.getwisdom.io/hacking-github-with-unicode-dotless-i/